citylearn/building.py

```diff
@@ -1653,6 +1802,10 @@ def next_time_step(self):
         self.dhw_storage.next_time_step()
         self.electrical_storage.next_time_step()
         self.pv.next_time_step()
+        if self.ev_chargers is not None:
```

Comment:
Fix the formatting please. White space should always precede the start of a conditional statement of for loop. Also, I think the default behavior should be that `ev_chargers` is an empty list and not `None`.